src/functions/lookupRandomCocktail.js

```diff
@@ -22,4 +22,4 @@ function lookupRandomCocktail (){
         });
 }
 
-module.exports = lookupRandomCocktail;
\ No newline at end of file
+export { lookupRandomCocktail };
```

Comment:
You can write this way too.
```
export function lookupRandomCocktail(...
```

Reply:
Thank you for your helpful suggestion!
I'll fix the code that way since it looks simpler and better!